src/main/java/mate/academy/model/Ticket.java

```diff
@@ -0,0 +1,78 @@
+package mate.academy.model;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.JoinColumn;
+import jakarta.persistence.ManyToOne;
+import jakarta.persistence.Table;
+import java.util.Objects;
+
+@Entity
+@Table(name = "Tickets")
+public class Ticket {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    @ManyToOne
+    private MovieSession movieSession;
+    @ManyToOne
+    @JoinColumn(name = "user_id")
+    private User user;
+
+    public Long getId() {
+        return id;
+    }
+
+    public Ticket setId(Long id) {
+        this.id = id;
+        return this;
+    }
```

Comment:
The `setId` method might not be necessary since the ID is auto-generated. Consider removing this setter to prevent accidental changes to the ID.